core/eolearn/core/utils/testing.py

```diff
@@ -80,3 +83,15 @@ def _get_feature_shape(
 ) -> Tuple[int, ...]:
     time, height, width, depth = len(timestamps), *config.raster_shape, rng.integers(*config.depth_range)
     return (time, height, width, depth) if ftype.is_temporal() else (height, width, depth)
+
+
+def assert_feature_match(tested_feature: Any, expected_feature: Any) -> None:
```

Comment:
change to `assert_feature_data_equal` or `assert_equal_features` or similar

Reply:
and add docstrings or pylint will eat you

Reply:
done